README.md

```diff
@@ -1,2 +1,70 @@
-# math-magicians
-A calculator made using REACT as frontend technology.
+# Getting Started with Create React App
```

Comment:
- [ ] I would recommend you do away with the dummy **`create-react-app`** texts that do not necessarily relate to the project in any way. I'd suggest you format your README using the Microverse template [here](https://github.com/microverseinc/readme-template)
- [ ] In your README remember to add instructions on how users can get your app repo locally, and also to run `npm install` to install packages and `npm start` to start project.

Reply:
Thanks, @TedLivist  for your feedback!, I really appreciate it. Your suggestions have been implemented on commit f154032.